presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -81,6 +91,21 @@ public void copyFileToContainer(Transferable transferable, String containerPath)
         copyFileToContainer(containerPath, () -> super.copyFileToContainer(transferable, containerPath));
     }
 
+    public DockerContainer withExposedLogPaths(String... logPaths)
+    {
+        this.logPaths.addAll(Arrays.asList(logPaths));
+        return this;
+    }
+
+    public void exposeLogsInHostPath(Path hostBasePath)
+    {
+        for (String containerLogPath : logPaths) {
+            Path hostLogPath = Paths.get(hostBasePath.toString(), containerLogPath);
+            cleanOrCreateHostPath(hostLogPath);
```

Comment:
I'd rather see caller do the cleanup (if needed)

This configuration method should ideally be just that -- a configuration method without side effects